Detectors/ITSMFT/ITS/simulation/src/V3Layer.cxx

```diff
@@ -524,11 +526,23 @@ TGeoVolume *V3Layer::createModuleInnerB(Double_t xmod, Double_t ymod, Double_t z
 {
   Double_t xtot, ytot, ztot, zchip;
   Double_t xpos, ypos, zpos;
-  char volumeName[30];
+  Bool_t dummyChip;
+  char chipName[30], sensName[30], volumeName[30];
+
+  // For material budget studies
+  if (mBuildLevel < 6)
+    dummyChip = kFALSE;  // will be made of Si
+  else
+    dummyChip = kTRUE;   // will be made of Air
 
   // First create the single chip
+  snprintf(chipName, 30, "%s%d", GeometryTGeo::getITSChipPattern(), mLayerNumber);
+  snprintf(sensName, 30, "%s%d", GeometryTGeo::getITSSensorPattern(), mLayerNumber);
+
   zchip = zmod / sIBChipsPerRow;
-  TGeoVolume *chipVol = createChip(xmod, ymod, zchip);
+  TGeoVolume *chipVol = AlpideChip::createChip(xmod, ymod, zchip,
```

Comment:
Aren't xmod and zchip constants of the Alpide? In principle, the mSensorThickness also, but probably we can have it settable in case we will use different Miko's response objects for chips of different total thickness

Reply:
@shahor02 I took out the X and Z parameters and made those dimensions fixed in AlpideChip.h, please check if this is ok. I left the sensor thickness as a parameter (at least, for the time being) because you said we may try to use different sensor thicknesses for IB and OB.